### DISCUSSION TITLE
Control ventilator

### DISCUSSION THREAD

**Discussion by folotp** [d0n4v4nb3ck3r/homebridge-honeywell-home#794] [Ideas]
Hello,

Would it be possible to add control of the Ventilator (not the HVAC fan) such as a heat-recovery ventilator (HRV).

Thanks.

**Comment by donavanbecker** [comment]
> Hello,
> 
> 
> 
> Would it be possible to add control of the Ventilator (not the HVAC fan) such as a heat-recovery ventilator (HRV).
> 
> 
> 
> Thanks.

Is it in the API?
